src/components/Confetti/Confetti.js

```diff
@@ -0,0 +1,12 @@
+const props = {
+  confettiParticles: {
```

Comment:
(nit) please simplify this prop name to: `particles` (the `confettti` prefix is redundant here since it's already obvious from the element/component name).